src/repos.js

```diff
@@ -4,9 +4,22 @@ const api = require('./api');
 const blacklist = ['rfcs'];
 
 const getForOrg = async org => {
-  const {data} = await api.repos.getForOrg({
-    org: org,
-  });
+  let pageNum = 1;
+  let results = [];
+  let isEmpty = false;
+  while (!isEmpty) {
+    const {data} = await api.repos.getForOrg({
```

Comment:
Defaults to [30 results per page](https://octokit.github.io/rest.js/#api-Issues-getForOrg) and we now have over 30 repositories in the `fusionjs` org.